app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+
+    for character in phrase:
+        if character.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
You could implement this task in one line using built-in string methods and chaining

Reply:
I understand, in the future I will be more practical in the approach and implementation of tasks 
Thank you